package.json

```diff
@@ -45,6 +45,7 @@
     "react-native-blob-util": "^0.13.18",
     "react-native-camera-kit": "^8.0.4",
     "react-native-config": "git+https://github.com/status-im/react-native-config.git#refs/tags/v1.4.2-status",
+    "react-native-countdown-circle-timer": "^3.0.9",
```

Comment:
Well, I use import this to avoid implement in our code. And, yes, this can be implemented in our code, it's using react native svg.
So, Should I implement this in our code and get rid of this dependency?

Reply:
done